Tools/ardupilotwaf/px4.py

```diff
@@ -188,12 +188,16 @@ def _px4_taskgen(bld, **kw):
 @feature('_px4_romfs')
 def _process_romfs(self):
     bld = self.bld
-    file_list = (
+    file_list = [
         'init.d/rc.APM',
         'init.d/rc.error',
         'init.d/rcS',
         (bld.env.PX4_BOOTLOADER, 'bootloader/fmu_bl.bin'),
-    )
+    ]
+
+    if bld.env.PX4_BOARD_RC:
+        board_rc = 'init.d/rc.%s' % bld.env.get_flat('PX4_BOARD_NAME')
```

Comment:
Yep... Take a look on the commit message on this one. It details the 3 levels of overrides  possible